src/replication.c

```diff
@@ -1574,7 +1574,14 @@ void sendBulkToSlave(connection *conn) {
     }
 
     /* If the preamble was already transferred, send the RDB bulk data. */
-    lseek(slave->repldbfd,slave->repldboff,SEEK_SET);
+    if (lseek(slave->repldbfd,slave->repldboff,SEEK_SET) == -1) {
+	serverLog(LL_WARNING,"Failed to lseek the RDB file to offset %lld for replica %s: %s",
+	    (long long)slave->repldboff,
+	    replicationGetSlaveName(slave),
+	    strerror(errno));
```

Comment:
since the line is not too long, how about keeping them in a single line
```suggestion
	serverLog(LL_WARNING,"Failed to lseek the RDB file to offset %lld for replica %s: %s",
	    (long long)slave->repldboff, replicationGetSlaveName(slave), strerror(errno));
```